src/views/lyrics.vue

```diff
@@ -394,10 +404,8 @@ export default {
       }, 50);
     },
     formatLine(line) {
```

Comment:
remove it? i not found call

Reply:
Seems like it has been existed for a long time. Are you sure that it is unused anywhere?